tools/generator/Tests/Integration-Tests/BaseGeneratorTest.cs

```diff
@@ -14,7 +14,7 @@ public class BaseGeneratorTest
 		StringWriter sw = null;
 
 		[SetUp]
-		public void Setup ()
+		public void SetUp ()
```

Comment:
Why the casing changes?

Reply:
I usually try to make these match the NUnit attribute, but doesn't have to.